tests/test_cram.py

```diff
@@ -271,5 +272,26 @@ def test_read_cram_file_definition(self):
             cram_file_definition = cram.read_fixed_length_cram_file_definition(f)
         self.assertEqual(cram_file_definition, expected, f'{cram_file_definition} is not: {expected}')
 
+    def test_samtools_prints_stderr_exception(self):
```

Comment:
Please remove this test, which is exercising functionality already tested in `tests/test_misc_utils.py`.